resources/quality/xyzprinting/xyzprinting_da_vinci_jr_pro_xeplus_global_0.40_coarse.inst.cfg

```diff
@@ -0,0 +1,27 @@
+[general]
+version = 4
+name = Coarse Quality
+definition = xyzprinting_da_vinci_jr_pro_xeplus
```

Comment:
This profile is exactly the same as the one for the other printer. Why duplicate it? You can just create a single profile and use the definition = xyzprinting_base

Reply:
hi, the file is a base for the printer's quality

Reply:
No, it is a global quality profile. It's not used by other quality profiles, just matched by the `quality_type`. Adding it to the parent printer `xyzprinting_base` allows it to be selected just as well.